frontend/src/features/landing_page/components/GeoMap.tsx

```diff
@@ -47,4 +50,29 @@ export const GeoMap = () => {
       {markers}
     </MapContainer>
   );
+
+  function handleClick() {
+    setIsVisible(true);
+  }
+
+  let content;
+
+  if (isVisible) {
+    content = map;
+  } else {
+    content = (
+      <div className="text-center" id="placeholder">
+        <img
+          src="/permaplant-map-placeholder.png"
+          alt="PermaplanT Map"
+          className="mb-2 hover:cursor-pointer"
```

Comment:
```suggestion
      <div className="flex h-full w-full flex-col text-center" id="placeholder">
        <img
          src="/permaplant-map-placeholder.png"
          alt="PermaplanT Map"
          className="mb-2 h-full w-full object-cover hover:cursor-pointer"
```

Making the img span the whole container and adding `object-size: cover` enhances the image on smaller screens.
It crops the edges and places it in the center. Without shrinking it down.
The flex container is so that the span below can grow to the size it needs and the img shrinks a little.

Reply:
This is my first experience with Tailwind, so I still don't know fully when and where I can and should add what classes.
But thank you for the tips, I will try to also think of them in future work.